goverge/main.py

```diff
@@ -58,6 +58,19 @@ def get_project_package(project_root, project_import):
     return project_import.replace("'", "")
 
 
+def get_test_packages(project_root):
+    """
+    :type project_root: string
+    :param project_root: The location of the project root
+    :rtype: List
+    :return: The list of to run coverage on
```

Comment:
It should have "packages" so "the list of packages to run coverage on". I will fix it quick.